build-constraints.yaml

```diff
@@ -3323,6 +3323,7 @@ packages:
 
     "Kowainik <xrom.xkov@gmail.com> @ChShersh":
         - base-noprelude == 4.11.1.0
+        - base-noprelude == 4.12.0.0
```

Comment:
Please remove the previous line as well.

Reply:
@snoyberg Done :+1: